platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gui/SwirldsGui.java

```diff
@@ -45,15 +45,19 @@ private SwirldsGui() {}
      *
      * @param platform the platform to create the console with
      * @param visible  should the window be initially visible? If not, call setVisible(true) later.
+     * @param localNodes the set of nodes running locally need content rendered in the gui.
      * @return the new window
      */
-    public static Console createConsole(final Platform platform, final int winNum, final boolean visible) {
+    public static Console createConsole(
+            @NonNull final Platform platform,
+            final int winNum,
+            final boolean visible,
+            @NonNull final Set<NodeId> localNodes) {
```

Comment:
You repeated for all calls the
```
platform.getRoster().rosterEntries().stream()
                        .map(RosterEntry::nodeId)
                        .map(NodeId::of)
                        .collect(Collectors.toSet()
```
maybe the parameter should be removed and called here internally?

Reply:
Using the config converter removes this.